src/composables/useFormSpaceSettings.ts

```diff
@@ -59,7 +59,10 @@ const formSettings = ref(clone(EMPTY_SPACE_FORM));
 const initialFormState = ref(clone(EMPTY_SPACE_FORM));
 const inputRefs = ref<any[]>([]);
 
-export function useFormSpaceSettings(context: 'setup' | 'settings') {
+export function useFormSpaceSettings(
+  context: 'setup' | 'settings',
+  { spaceType = 'default' } = {}
```

Comment:
Same here, why do we need `= {}`?

Reply:
There are some places where we don't pass these params to `useFormSpaceSettings` or `useFormSpaceProposal` composables. It is a bit painful to pass `spaceType` everywhere. 🙈 so this param will be `{}` by default.

Reply:
But you already defined it here to be `default` always with `spaceType = 'default'`

Reply:
If I'm not mistaken the object can never be just `{}`

Reply:
Ok checked and seems to be needed. Still not sure I understand why tho :D

Reply:
Yea bit confusing 😄